apps/web/assets/js/components/Opportunity.jsx

```diff
@@ -26,7 +26,7 @@ export default class Opportunity extends React.Component {
                 <a href={project.url}>{project.name}</a> 
                 <span className="opportunity__title-icon">&#9655;</span>
               </span>
-              <a href={url}>{title}</a> 
+              <a href={project.url}>{project.title}</a> 
```

Comment:
I _think_ this is incorrect.  This should be the opportunities `url` and `title`, not the `project` which is above.

Reply:
Ohhh, ok. Do those exist elsewhere? Those two variables were undefined and throwing errors when I first booted the app

Reply:
Maybe they need to be destructured off a different part of props?

Reply:
Hmm 🤔

This is the pre-refactored code:

```javascript
 <a href={this.props.data.url}>{this.props.data.title}</a> 
```

I'll try it again locally 👍

Reply:
ahhhh, got it. It looks like the issue was actually just the `url` wasn't getting pulled out of `this.props.data`.

Fix incoming